ex04.ex

```diff
@@ -75,9 +75,15 @@ defmodule Ex04 do
   return value will be the thing you have to manipulate.
   """
 
-  def even_odd . . . "your code"
-
+  def _even_odd_tuple(x, { even, odd }) when Integer.is_even(x) do
```

Comment:
I'll let you get away with this because I never explicitly said it, but using ++ to build lists is actually quite expensive for larger lists, as it involves a full copy.

It's idiomatic to reverse either the input before processing or the output after.